test/suite/test_encrypt01.py

```diff
@@ -61,13 +61,20 @@ class test_encrypt01(wttest.WiredTigerTestCase):
         ('none-snappy', dict(log_compress=None, block_compress='snappy')),
         ('snappy-lz4', dict(log_compress='snappy', block_compress='lz4')),
     ]
-    scenarios = make_scenarios(types, encrypt, compress)
+    loadExt = [
+        ('earlyLoadTrue', dict(earlyLoad=True)),
```

Comment:
I thought we do not use camel casing for variable names. But else "skipIfMissing" in wttest.py proved me otherwise.

Reply:
@sulabhM Yeah, I used camel casing since the same format is used in the wttest.py file.